### DISCUSSION TITLE
Leaflet Version 2

### DISCUSSION THREAD

**Discussion by githubtilemaker** [Leaflet/Leaflet#9079] [Q&A] [answered]
I am just starting to plan a new project. I see that a version 2 is coming up.

Will there be many changes that may not be compatible with the current version?

What do you recommend when starting something new that will certainly not go live before the beginning of next year?

**Comment by jonkoops** [comment, marked as answer]
We will not be committing to anything related to Leaflet version 2 at this time as what will land is highly dependent on our capacity as a team. I'd recommend following the discussion under #7729.
